### PR TITLE
Gamedb: add patches to 2k games

### PR DESCRIPTION
**Pull request by Mrlinkwii**
### Description of Changes

add patches to various 2k games 
### Rationale behind Changes

fixes said games 
### Suggested Testing Steps

look at CI/ test said games

### CODE REVIEW

File: bin/resources/GameIndex.yaml [hunk 1]
```diff
@@ -13131,6 +13133,14 @@ SLES-53021:
 SLES-53022:
   name: "ESPN NBA 2K5"
   region: "PAL-E"
+  gameFixes:
+    - SkipMPEGHack #Fixes videos.
```
Comment (refractionpcsx2) [inline bin/resources/GameIndex.yaml:13137]: missing space after #

File: bin/resources/GameIndex.yaml [hunk 2]
```diff
@@ -9502,6 +9502,8 @@ SLES-51145:
 SLES-51150:
   name: "Scrabble Interactive - 2003 Edition"
   region: "PAL-M3"
+  clampModes:
+    vuClampMode: 0 # Fixes scrabble board issue.
```
Comment (refractionpcsx2) [inline bin/resources/GameIndex.yaml:9506]: It's more generically "missing UI and game elements."

**Comment by lightningterror** [comment]
Commit could've been named better.

**Comment by refractionpcsx2** [comment]
I know, I didn't realise until I merged.

**Pull request merged**